frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationTask.tsx

```diff
@@ -203,27 +201,28 @@ const TaskComponent: React.FC<TaskProps> = ({
         renderVisualName
       )}
 
-      {isPipelineRun && showStatusState && (
-        <g
-          className={cx({
-            'fa-spin odc-pipeline-vis-task--icon-spin': status.reason === runStatus.Running,
-            'odc-pipeline-vis-task--icon-stop': status.reason !== runStatus.Running,
-          })}
-        >
-          <svg
-            width={30}
-            height={30}
-            viewBox="-5 -4 20 20"
-            style={{
-              color: taskStatusColor,
-            }}
-          >
-            <StatusIcon status={status.reason} disableSpin />
-          </svg>
-        </g>
-      )}
       {showStatusState && (
-        <SvgTaskStatus steps={stepStatusList} x={30} y={23} width={width / 2 + 15} />
+        <>
```

Comment:
FYI @karthikjeeyar / @debsmita1  This will be a conflict with https://github.com/openshift/console/pull/9520

Reply:
Rebased my PR